web_ui/frontend/app/cache/page.tsx

```diff
@@ -0,0 +1,142 @@
+/***************************************************************
+ *
+ * Copyright (C) 2023, Pelican Project, Morgridge Institute for Research
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ***************************************************************/
+
+"use client"
+
+import {Box, Grid, Typography} from "@mui/material";
+
+import RateGraph from "@/components/graphs/RateGraph";
+import StatusBox from "@/components/StatusBox";
+import {DataExportTable} from "@/components/DataExportTable";
+import {TimeDuration} from "@/components/graphs/prometheus";
+import FederationOverview from "@/components/FederationOverview";
+import LineGraph from "@/components/graphs/LineGraph";
+
+export default function Home() {
+
+    return (
+        <Box width={"100%"}>
+            <Grid container spacing={2}>
+                <Grid item xs={12} lg={4}>
+                    <Typography variant="h4">Status</Typography>
+                    <StatusBox/>
+                </Grid>
+                <Grid item xs={12} lg={4}>
+                    <Typography variant={"h4"} component={"h2"} mb={2}>Data Exports</Typography>
+                    <Box sx={{borderRadius: 2, overflow: "hidden"}}>
+                        <DataExportTable/>
+                    </Box>
+                </Grid>
+                <Grid item xs={12} lg={4}>
+                    <FederationOverview/>
+                </Grid>
+                <Grid item xs={12} lg={6}>
+                    <Box sx={{backgroundColor: "#F6F6F6", borderRadius: "1rem"}} p={2}>
+                        <Typography variant="h4" mb={1}>Storage</Typography>
+                        <Box minHeight={"200px"}>
+                            <LineGraph
```

Comment:
One other thing I noticed is that the time format is a bit weird: 

![Screenshot 2024-04-03 at 4 09 02 PM](https://github.com/PelicanPlatform/pelican/assets/41393704/0d3bce0b-8cc2-49a9-b904-d00a559e55e3)

The second-level precision should be good enough for our users. We should fix this if it's an easy fix